docs/manual/forms/translations.rst

```diff
@@ -0,0 +1,107 @@
+===========
+
+.. TODO Not sure if Vertalingen ingeschakeld is the correct literal, as it isn't translated yet
+
+Open Formulieren biedt de mogelijkheid om formulieren aan te bieden in meerdere talen (momenteel alleen Nederlands en Engels).
+Om dit mogelijk te maken is het van belang dat de Nederlandse termen in het formulier vertaald worden naar de gekozen taal
+en in deze handleiding wordt uitgelegd hoe deze vertalingen in te voeren zijn via de beheerinterface.
+
+Globale configuratie vertalingen
+================================
+
+.. TODO Several labels mentioned in the enumeration not yet translated
+
+Allereerst zijn er een aantal globaal gebruikte woorden en zinnen waarvan de vertalingen ingevoerd moeten worden,
+dit kan door naar **Configuratie > Algemene configuratie** te navigeren. Op die pagina zijn een aantal velden te zien
+met een **[nl]** en een **[en]** versie. Op deze manier is het mogelijk om voor de volgende velden vertalingen in te voeren:
+
+    - **Inzendingen > Bevestigingspagina tekst**
+    - **Bevestigingse-mail > Onderwerp**
+    - **Bevestigingse-mail > Inhoud**
+    - **Sjabloon "pauzeer"-email > Onderwerp**
+    - **Sjabloon "pauzeer"-email > Inhoud**
+    - **Knop labels > Formulier starten-label**
+    - **Knop labels > Back to form text**
+    - **Knop labels > Stap wijzigen-label**
+    - **Knop labels > Formulier verzenden-label**
+    - **Knop labels > Previous step text**
+    - **Knop labels > Opslaan-label**
+    - **Knop labels > Volgende stap-label**
+    - **Privacy > Privacybeleid label**
+
+    .. image:: _assets/translations_global_config.png
+        :width: 100%
+
+Formulier vertalingen
+=====================
+
+Formulierdetails
+----------------
+
+De volgende stap is om formulierspecifieke vertalingen in te voeren, dit gebeurt in de beheerinterface van het desbetreffende formulier.
+
+Allereerst moeten de vertalingen voor de naam en het toelichtingssjabloon ingevoerd worden. Het wisselen tussen talen gebeurt d.m.v.
+de tabjes boven de velden.
+
+    .. image:: _assets/translations_form_details.png
+        :width: 100%
+
+Stappen en velden
+-----------------
+
+Bij de stapgegevens zijn er een aantal vertalingen die ingevoerd moeten worden, zoals te zien hieronder. Wisselen tussen talen
+gebeurt wederom met de tabjes. Indien deze niet ingevuld zijn, worden de standaardvertalingen uit de algemene configuratie gebruikt
+
+    .. image:: _assets/translations_formstep.png
+        :width: 100%
+
+.. TODO UI in these images might be subject to change, in the future they will probably be prefilled
+
+De vertalingen voor de velden van het formulier worden ingevoerd per veld, door te klikken op het tandwiel-icoontje
+van het veld. Vervolgens kunnen vertalingen ingevoerd worden bij de tab **Vertalingen**:
+
+    .. image:: _assets/translations_formio.png
+        :width: 100%
+
+Vertalingen worden hergebruikt per stap, dus als twee velden hetzelfde label hebben, zullen ze ook dezelfde vertaling krijgen.
```

Comment:
Probably best to put this in a `.. note::` directive so it's highlighted/stands out more